docs/debugging.md

```diff
@@ -6,8 +6,8 @@ issues.
 
 ## Backtrace
 
-If you're seeing a failure nested somewhere deep in an application but can't 
-work out how it's getting there, you can use the `__faasm_backtrace` function 
+If you're seeing a failure nested somewhere deep in an application but can't
+work out how it's getting there, you can use the `__faasm_backtrace` function
```

Comment:
You'll have to ignore the changes to the docs files as it's almost all whitespace trimming apart from swapping `simple_runner` for `func_runner`